ansible/deploy_etckeeper.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Put /etc under version control using etckeeper
+  hosts: all
+  become: True
+  roles:
+    - role: debops-contrib.etckeeper
+      tags: 'etckeeper'
```

Comment:
should this be referenced in `deploy_stack` so that it always gets deployed? or do you only want it only deployed when specified?